projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -445,6 +450,38 @@ private function load_subtitle( $task ) {
 		return '';
 	}
 
+	/**
+	 * Loads the extended content for a task.
+	 *
+	 * @param Task $task A task definition.
+	 * @return array|null The extended content for the task.
+	 */
+	private function load_extended_content( $task ) {
+		if ( ! isset( $task['extended_content'] ) ) {
```

Comment:
We should also check that `$task['extended_content']` is an array before we `foreach` over it.
```suggestion
		if ( ! isset( $task['extended_content'] ) || ! is_array( $task['extended_content'] ) ) {
```